styles.css

```diff
@@ -5,13 +5,12 @@
 .shoe4 { height: 100px; width: 180px; }
 .shoppingcart { width: 15px; }
 h1 { color: red; font-family: 'PT Sans Narrow', sans-serif;  font-size: 10px;  }
-ul { display: inline-block; }
-li { display: inline-block; color: red;  margin: 10px; }
-
-nav { position: relative;  }
+nav { position: relative; }
 footer { color: red; font-family: 'PT Sans Narrow', sans-serif;  font-size: 10px; }
 ul { display: inline-block; }
 li { display: inline-block; color: red;  margin: 10px; }
 .left { width: 50%; left: 5px; text-align: left; }
 .right { width: 40%; right: 50px; text-align: right;}
-.shoe-background { background: linear-gradient(to right, purple, red);float: center; display: block; width: 100% height: 300px; }
+.shoe-background { background: linear-gradient(to right, purple, red); float: center; display: block; width: 100%; }
+.short { height: 150px; }
+.tall { height: 300px; }
```

Comment:
These two new classes (`.short` and `.tall`) were added so you didn't have two 300px tall spaces on the page. You can adjust these however you'd like! 😄